### PR TITLE
DDF File for Xiaomi Mijia Wireless Switch WXKG01LM

### PR DESCRIPTION
**Pull request by mattreim**
Xiaomi Mijia Wireless Switch WXKG01LM:

Xiaomi Mijia Wireless Switch WXKG01LM

Log:

wxkg01lm

### CODE REVIEW

File: devices/xiaomi/xiaomi_wxkg01lm_mini_switch.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/battery",
+          "awake": true,
+          "parse": { 
+            "at": "0xFF02",
+            "ep": 1,
+           "fn": "xiaomi:special",
```
Comment (SwoopX) [inline devices/xiaomi/xiaomi_wxkg01lm_mini_switch.json:78]: Could you please correct the indent here? Thx!

**Review by SwoopX — CHANGES_REQUESTED** [review]
Hey,

thanks a lot for sharing a DDF for a rather old device! We could definitively use more like these to get rid of the legacy heritage, so this is highly appreciated!

Apart from the small blemish, I guess this one looks good to go. Not too sure about the firmware version part, but it doesn't hurt anyway 👍

File: devices/xiaomi/xiaomi_wxkg01lm_mini_switch.json (hunk 1 above)
Comment (mattreim) [reply devices/xiaomi/xiaomi_wxkg01lm_mini_switch.json]: Corrected

Comment (mattreim) [reply devices/xiaomi/xiaomi_wxkg01lm_mini_switch.json]: Sorry

**Comment by SwoopX** [comment]
Erm, not sure why we ended up here but now things feel a bit mixed up 😅 

Maybe you want to restart with detangled PR(s)?

**Pull request closed without merging**